doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -259,39 +269,45 @@ Skia-based targets provide support for full XAML Hot Reload and C# Hot Reload. T
 - Rider
   - Hot Reload is only supported without the debugger.
-  - Adding new C# or XAML files to a project is not yet supported
+  - Adding new C# or XAML files to a project is not yet supported.
 
 ### [**WebAssembly**](#tab/wasm)
 
-WebAssembly is currently providing both full and partial Hot Reload support, depending on the IDE.
+WebAssembly is currently providing full Hot Reload support.
 
 - In Visual Studio Code:
-  - Both C# and XAML Hot Reload are fully supported
-  - Adding new C# or XAML files to the project is not yet supported
-  - Hot Reload is not supported when using the debugger
+  - Both C# and XAML Hot Reload are fully supported.
+  - Adding new C# or XAML files to the project is not yet supported.
+  - Hot Reload is not supported when using the debugger.
 - In Rider:
-  - Both C# and XAML Hot Reload are fully supported
-  - Adding new C# or XAML files to the project is not yet supported
-  - Hot Reload is not supported when using the debugger
-- In Visual Studio for Windows:
-  - [`MetadataUpdateHandlers`](https://learn.microsoft.com/dotnet/api/system.reflection.metadata.metadataupdatehandlerattribute) are invoked without the list of changed types, which means that some hot reload features may not be available. This feature is slated to be available for .NET 9.
+  - Both C# and XAML Hot Reload are fully supported.
+  - Adding new C# or XAML files to the project is not yet supported.
+  - Hot Reload is not supported when using the debugger.
+
+### [**iOS, Android**](#tab/mobile)
+
+Mobile targets now support both XAML and C# Hot Reload. Debugger-specific variations apply depending on the IDE.
+
+- In Visual Studio:
+  - The debugger **has** to be attached.
+- In VS Code, and Rider:
+  - Hot Reload is not supported when using the debugger.
+- XAML `x:Bind` Hot Reload is limited to simple expressions and events.
 
-### [**iOS, Android, and Catalyst**](#tab/mobile)
+### [**Catalyst**](#tab/catalyst)
 
-Mobile targets are currently using a limited version of XAML Hot Reload and do not support C# Hot Reload until [this dotnet runtime](https://github.com/dotnet/android/issues/9120) issue is fixed.
+Mobile targets now support both XAML and C# Hot Reload.
 
-- In Visual Studio, the "Hot Reload on File Save" feature must be disabled to avoid crashing the app. You can find this feature by clicking on the down arrow next to the red flame in the Visual Studio toolbar.
-- In VS, VS Code, and Rider, [C# Hot Reload is not yet fully supported](https://developercommunity.visualstudio.com/t/net70-iosnet70-android-MetadataUpd/10279604#T-ND10384434)
-- XAML `x:Bind` hot reload is limited to simple expressions and events
+- XAML `x:Bind` hot reload is limited to simple expressions and events.
 
 ### [**WinAppSDK**](#tab/winappsdk)
 
 - Hot Reload is supported by Visual Studio for WinAppSDK and provides support in unpackaged deployment mode.
-- Hot Reload is not supported in Rider
+- Hot Reload is not supported in VS Code and Rider.
 
 ---
 
```

Comment:
@jeromelaban, @dr1rrb I think some updates are also needed for the **Troubleshooting** section per IDE:
https://github.com/unoplatform/uno/blob/271a65009818f0532fe8dca05e7454361191dc9f/doc/articles/features/working-with-xaml-hot-reload.md?plain=1#L338-L417

Reply:
Following updates for this will be tracked with: https://github.com/unoplatform/uno/issues/19353